src/components/transactions/messageTypes.js

```diff
@@ -25,4 +25,15 @@ const transactionGroup = {
   [messageType.WITHDRAW_DELEGATION_REWARD]: "distribution"
 }
 
-export { messageType, transactionGroup }
+const getTransactionGroup = group => {
+  if (typeof transactionGroup[group] === "undefined") {
+    const groups = group.split("/")
+    if (groups.length > 1) {
+      return transactionGroup[groups[1]]
+    }
+    return false
+  }
+  return transactionGroup[group]
+}
+
+export { messageType, transactionGroup, getTransactionGroup }
```

Comment:
Why do you add this here if we are not using it anywhere?

Maybe another PR?

Reply:
It was used in the extension. For some reasons it was removed from this file, but not from the callee

Reply:
Ah, should have guessed it. Cool :+1: